packages/core/src/client/create-client.ts

```diff
@@ -126,7 +127,13 @@ export const createPublicErc4337Client = ({
   const client = createPublicErc4337FromClient(
     createPublicClient({
       chain,
-      transport: http(rpcUrl),
+      transport: http(rpcUrl, {
+        fetchOptions: {
+          headers: {
+            "Alchemy-AA-Sdk-Version": VERSION,
```

Comment:
Since we're using `viem`, we can only specify headers globally rather than per request, so we can't send the method being called like the current alchemy SDK does. That should be fine though because we'll have that info anyways